curbmap-ios-mvp/Services/AuthServices.swift

```diff
@@ -19,10 +19,10 @@ let AUTH_HOST = "https://curbmap.com"
  */
 func login(callback: @escaping (_ result: Int)->Void) -> Void {
     let parameters = [
-        "username": currentUser.getUsername(),
-        "password": currentUser.getPassword()
+        "username": User.currentUser.getUsername(),
+        "password": User.currentUser.getPassword()
     ]
-    if (currentUser.getUsername() == "" || currentUser.getPassword() == "") {
+    if (User.currentUser.getUsername() == "" || User.currentUser.getPassword() == "") {
```

Comment:
For future refactoring: user input validation should be handled at view controller level

Reply:
This is a secondary test, not to take the place of true validation. I mean, we can take it out because it happens on the backend also, but I thought it would necessarily save an extra data round trip